common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilter.java

```diff
@@ -195,13 +195,14 @@ private static int optimalNumOfHashFunctions(long n, long m) {
   /**
    * Computes m (total bits of Bloom filter) which is expected to achieve, for the specified
    * expected insertions, the required false positive probability.
-   *
-   * See http://en.wikipedia.org/wiki/Bloom_filter#Probability_of_false_positives for the formula.
+   * <p>
+   * See <a href="http://en.wikipedia.org/wiki/Bloom_filter#Probability_of_false_positives">
```

Comment:
The change in comments is to fix the `unidoc` check failure, because after this method becomes public, it will be involved in the `unidoc` check.